tardis/energy_input/GXPacket.py

```diff
@@ -79,6 +79,32 @@ def get_location_r(self):
         )
 
 
+class GXPacketCollection:
+    def __init__(
+        self,
+        location,
+        direction,
+        energy_rf,
+        energy_cmf,
+        nu_rf,
+        nu_cmf,
+        status,
+        shell,
+        time_current,
+    ):
+        self.location = location
+        self.direction = direction
+        self.energy_rf = energy_rf
+        self.energy_cmf = energy_cmf
+        self.nu_rf = nu_rf
+        self.nu_cmf = nu_cmf
+        self.status = status
+        self.shell = shell
+        self.time_current = time_current
```

Comment:
What is `time_current`?

Reply:
The current local time value of the packet, if there was a clock attached to it